module/multimedia/src/Application/Service/DuplicateFilenameSuffixGeneratingService.php

```diff
@@ -13,13 +13,18 @@ class DuplicateFilenameSuffixGeneratingService implements SuffixGeneratingServic
 {
     private const MAX_LENGTH = 128;
 
-    public function generateSuffix(string $name, int $iterationIndex): string
+    public function generateSuffix(string $filename, string $extension, int $iterationIndex): string
```

Comment:
Basically, it seems like the extension is redundant from the service point of view. Perhaps worth ommiting - it's not really its responsibility to concatenate the full name at the end?